client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -51,6 +52,27 @@ module.exports = React.createClass( {
 		} );
 	},
 
+	monthlyPrice: function() {
+		const { cost, product_slug: productSlug, currency } = this.props.cartItem,
+			// A hacky way to check if we are in signup, since for some reason cartItem.extra.context
+			// is missing in that case (it seems cartItem.extra is overwritten somewhere between)
+			isInSignup = this.props.cartItem.extra && this.props.cartItem.extra.context !== 'calypstore',
+			isPaidPlan = productSlug === 'value_bundle' || productSlug === 'business-bundle';
```

Comment:
Awesome idea! :+1: Should have remembered about those functions...
And yes, they do work with cart items (the only required field for the "product" is `product_slug`/`productSlug` and we have that).